src/index.html

```diff
@@ -1,13 +1,80 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0
+      "
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="stars.css">
+    <link rel="stylesheet" href="product.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        class="product-card__photo"
+        src="./images/imac.jpeg"
+        alt="APPLE iMac photo"
+      >
+
+      <div class="product-card__title">
+        <h2 class="product-card__title-text">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h2>
+      </div>
+
+      <div class="product-card__code">
+        <p class="product-card__code-product">
+          Код товара: 195434
+        </p>
+      </div>
+
+      <div class="product-card__raiting">
+        <div class="stars stars--5">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="product-card__raiting-comments">
+          Отзывов: 5
+        </p>
+      </div>
+
+      <div class="product-card__price">
+        <p class="product-card__price-card">
+          Цена:
+        </p>
+
+        <p class="product-card__price-card_price">
```

Comment:
```suggestion
        <p class="product-card__price-value">
```
let's try to make naming less complicated